envoy.code.check/envoy/code/check/abstract/__init__.py

```diff
@@ -2,20 +2,24 @@
 from .base import ACodeCheck
 from .checker import ACodeChecker
 from .flake8 import AFlake8Check
+from .shellcheck import AShellcheckCheck
```

Comment:
why do we need like this? checkCheck? :)

Reply:
by making the checks all of a standard - and inheriting from the base class it saves a lot repeated code.

also it makes it more extensible as we add more checks, ie not adding them actually to the checker, but maybe i misunderstand the question